.mise.toml

```diff
@@ -1,28 +1,84 @@
-[tasks."python:check"]
+[tasks."python:format"]
+description="format python code"
+run = "ruff format"
+
+[tasks."python:dead-code"]
+depends = ["python:format"]
+description="check for dead python code"
+run = """
+vulture \
+    --min-confidence 80 \
+    --exclude '.flox,.venv,target' \
+    .
+"""
+
+[tasks."python:lint"]
+depends = ["python:dead-code"]
 description="run python code quality checks across the full repository"
 run = """
-uvx ruff check \
+ruff check \
     --output-format=github \
     .
+"""
+
+[tasks."haskell:format"]
+description = "lint haskell code"
+run = "ormolu --mode inplace $(find . -name '*.hs')"
 
+[tasks."haskell:lint"]
+description = "lint haskell code"
+depends = ["haskell:format"]
+run = "hlint ."
+
+[tasks."haskell:build"]
+description = "build haskell code"
+run = "cd platform && stack build"
+
+[tasks."haskell:test"]
+description = "run haskell tests"
+run = "cd platform && stack test"
```

Comment:
It might be the case for the current configuration but with what I'm adding with the root `stack.yaml` there won't be any `cd`ing just a heads up.